go/runtime/scheduling/api/api.go

```diff
@@ -32,6 +39,9 @@ type Algorithm interface {
 	// IsQueued returns if a transaction is queued.
 	IsQueued(hash.Hash) bool
 
+	// RemoveTxBatch removes a transaction batch.
+	RemoveTxBatch(tx [][]byte) error
```

Comment:
Maybe name consistently, there is `ScheduleBatch`, `ScheduleTx` and `RemoveTxBatch`. Either add `Tx` everywhere or remove it from everywhere (no strong preference).